nano/node/wallet.cpp

```diff
@@ -1692,7 +1692,8 @@ void nano::wallets::ongoing_compute_reps ()
 {
 	compute_reps ();
 	auto & node_l (node);
-	auto compute_delay (network_params.network.is_dev_network () ? std::chrono::milliseconds (10) : std::chrono::milliseconds (15 * 60 * 1000)); // Representation drifts quickly on the test network but very slowly on the live network
+	// Representation drifts quickly on the test network but very slowly on the live network
+	auto compute_delay = network_params.network.is_dev_network () ? std::chrono::milliseconds (10) : (network_params.network.is_test_network () ? std::chrono::milliseconds (nano::test_scan_wallet_reps_delay ()) : std::chrono::minutes (15));
```

Comment:
Personally, I would test for NANO_TEST_WALLET_SCAN_REPS_DELAY here so that there are no 2 copies of the 15 minute delay. But it is a minor point.

Reply:
I don't like the magical constants either, it would be great if there was a single header with everything so then both functions would refer to a single variable